ports/nrf/boards/s140_7.2.0.ld

```diff
@@ -1,4 +1,4 @@
 /* GNU linker script for s140 SoftDevice version 6.1.1 */
```

Comment:
The comment should also be updated to reflect the new version.

Reply:
Will change it and do a new pull.  Thanks